Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/ISISIndirectDiffractionReduction.py

```diff
@@ -113,9 +122,13 @@ def validateInputs(self):
 
         # Validate input files
         input_files = self.getProperty('InputFiles').value
+
         if len(input_files) == 0:
```

Comment:
Should this not be part of the "invalid_number_range" function?

Reply:
They are checking for different problems so they should be separate, but I see why you think this as the name I gave the function `contains_invalid_number_range` is not clear so I have changed it to `contains_non_ascending_range`